package.json

```diff
@@ -53,7 +53,7 @@
     "page-icon": "^0.3.0",
     "progress": "^2.0.0",
     "sanitize-filename": "^1.5.3",
-    "shelljs": "^0.7.0",
+    "shelljs": "^0.7.8",
```

Comment:
@zucher alright to bump this dep if you need it, and thanks for adding `eslint-plugin-promise`. But if you modify `package.json` you need to regenerate, commit, and push [`package-lock.json`](https://docs.npmjs.com/files/package-lock.json) too. If you're not familiar with it, the TL;DR is: `rm -rf node_modules package-lock.json && npm install && git add package-lock.json && git commit -m "Regenerate package-lock.json"`

Reply:
Where come from package-lock.json, with npm install I don't generate the package-lock.json ?

Reply:
@zucher ah, you must be on an old version of `npm`. Don't worry about it, then, I'll do it. Keep me posted about the documentation, then I will get your branch, add a single commit for the package-lock thingie, and merge.

Reply:
Thank you